### PR TITLE
Add Indonesian language at base.php

### PR DESCRIPTION
**Pull request by vonsogt**
add more keyword for Indonesian language

### CODE REVIEW

**Comment by ziming** [comment]
You have 2 PRs on this, which should be the 1 checked?

**Comment by vonsogt** [comment]
> You have 2 PRs on this, which should be the 1 checked?

its different file

**Comment by ziming** [comment]
Alright, let me get my indonesian colleague to help check

**Comment by ziming** [comment]
Alright, will get my indonesian colleague to review this PR when he is free.

File: src/resources/lang/id/base.php [hunk 1]
```diff
@@ -8,27 +8,63 @@
...
-    'click_here_to_reset'  => 'Klik Disini untuk menyetel ulang kata sandi',
-    'unauthorized'         => 'Tidak Sah.',
-    'dashboard'            => 'Dashboard',
-    'handcrafted_by'       => 'Handcrafted by',
-    'powered_by'           => 'Powered by',
+    'registration_closed'    => 'Pendaftaran di tutup.',
+    'no_email_column'        => 'Pengguna tidak memiliki alamat email terkait.',
+    'first_page_you_see'     => 'Halaman pertama yang Anda lihat setelah login',
+    'login_status'           => 'Status Masuk',
+    'logged_in'              => 'Anda sudah login!',
```
Comment (stephanus-stuff) [inline src/resources/lang/id/base.php]: login / masuk

**Comment by tabacitu** [comment]
Great work here! Thank you @vonsogt , @ziming & @stephanus-stuff ! Merged and tagged. It's official - Backpack now has Indonesian translation thanks to you guys. 🎉

**Pull request merged**